### DISCUSSION TITLE
How to choose server requirement for 1080p, 2K, and 4K along with adaptive live streaming

### DISCUSSION THREAD

**Discussion by swatisinha-mis** [ant-media/Ant-Media-Server#5693] [Q&A]
We are developing live streaming web application where technical requirements are 1080p, 2K, and 4K along with adaptive for live streaming. We want good quality, high resolution along with bandwidth, Can you please suggest to us the server requirements and how to achieve this.
Please Find the attachment below of our current server detail.
![admindashboardss](https://github.com/ant-media/Ant-Media-Server/assets/99073345/c76d9c82-9b95-4e07-9ed8-719351d70734)

**Comment by lastpeony** [comment]
Hello,
What is your expected total number of viewers and streams?

Most probably you will need GPU instances because of adaptive bitrate streaming. 

Also you might need clustering to scale.
https://antmedia.io/docs/guides/clustering-and-scaling/scaling-ant-media-server/
